src/web/resources/report-page.js

```diff
@@ -299,31 +375,17 @@ function buildCompareForm(jsonData) {
         placeholder: "current report against"
     }, [])
 
-    var showToleranceBool = false
-    if (radioStates[radioStatesIndex] == "time" ||
-        radioStates[radioStatesIndex] == "targetAccuracy" ||
-        radioStates[radioStatesIndex] == "resultAccuracy" ||
-        radioStates[radioStatesIndex] == "startAccuracy") {
-        showToleranceBool = true
-    }
```

Comment:
You should make separate PRs contain independent code.

Reply:
Can do. Was fixing how the options. Is it common to have 4-6 pull request for this amount of changes?